elements/lisk-codec/src/json_wrapper.ts

```diff
@@ -15,15 +15,16 @@
 
 import {
 	BaseTypes,
-	GenericObject,
+	IteratableGenericObject,
 	SchemaPair,
 	SchemaProps,
 	SchemaScalarItem,
 } from './types';
 
-export interface IteratableGenericObject extends GenericObject {
+interface iteratorReturnValue {
```

Comment:
Types always capitalize. 

```suggestion
interface IteratorReturnValue {
```